src/Microsoft.OData.Core/ODataCollectionWriterCore.cs

```diff
@@ -266,9 +268,20 @@ void IODataOutputInStreamErrorListener.OnInStreamError()
         }
 
         /// <inheritdoc/>
-        Task IODataOutputInStreamErrorListener.OnInStreamErrorAsync()
+        async Task IODataOutputInStreamErrorListener.OnInStreamErrorAsync()

```

Comment:
This method is private?
If so, I would prefer it to have the `private` modifier

Reply:
@KenitoInc The `IODataOutputInStreamErrorListener` interface is implemented explicitly. Applying a `private` modifier or even a `public` modifier is not the right thing to do nor is it required.
Based on [C# specification](https://www.microsoft.com/en-us/download/details.aspx?id=7029):
_Explicit interface member implementations have different accessibility characteristics than other members. Because explicit interface member implementations are never accessible through their fully qualified name in a method invocation or a property access, they are in a sense private. However, since they can be accessed through an interface instance, they are in a sense also public._

It's accessible through the interface instance as shown below so it's not quite private:
```csharp
public interface I
{
    void M();
}

public class C : I
{
    // Explicit implementation
    void I.M();
}

I i = new C();
i.M();
```